Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonDataGridView.cs

```diff
@@ -2912,7 +2912,11 @@ private void OnContextMenuStripOpening(object? sender, CancelEventArgs e)
     /// </summary>
     [Browsable(false)]
     [EditorBrowsable(EditorBrowsableState.Advanced)]
-    public ToolStripRenderer CreateToolStripRenderer() => Renderer?.RenderToolStrip(GetResolvedPalette()!)!;
+    public ToolStripRenderer CreateToolStripRenderer()
+    {
+        var palette = GetResolvedPalette() ?? KryptonManager.CurrentGlobalPalette;
+        return Renderer?.RenderToolStrip(palette)!;
```

Comment:
This can still return null, So probably easier just to make the API nullable.
Like you have done with the rest.